packages/msgraph/src/Adaptor.js

```diff
@@ -23,8 +29,14 @@ export function execute(...operations) {
   };
 
   const cleanup = finalState => {
-    const { drives, ...rest } = finalState;
-    return rest;
+    if (finalState?.buffer) {
```

Comment:
I actually prefer the old pattern here:
```
const { drives, buffer,  ... rest } = finalState;
return rest;
```

Except actually I'd do like this:

```
const cleaup = (state) => {
  const { drives, buffer,  ... finalState } = state;
  return finalState;
}
```

Reply:
@josephjclark , that looks clean but you will into an error saying `can not destructure undefine`  if `drives` or `buffer` is not in state. That why i had to check if i have the key `buffer` or `drives` in state before removing it